back/app/models/custom_field.rb

```diff
@@ -317,6 +313,15 @@ def linear_scale_print_description(locale)
     end
   end
 
+  def input_term
+    phase = if resource.participation_context.instance_of?(Project)
+      TimelineService.new.current_or_backup_transitive_phase(resource.participation_context)
+    else
+      resource.participation_context
+    end
+    phase&.input_term || Phase::DEFAULT_INPUT_TERM
```

Comment:
Great! Can be done later but when we create a proposal phase, the default input term should be "proposal", instead of "idea".

Reply:
That's a good point! Perhaps the default input term could be delegated to the participation method. I added this as a point to the input term ticket.